index.d.ts

```diff
@@ -566,6 +566,10 @@ declare module 'plaid' {
     categories: Array<Category>;
   }
 
+  interface CreateItemAddTokenResponse extends BaseResponse {
+    add_token: string;
```

Comment:
Wondering if we should also pass back the expiry since that's one of the fields listed in the Plaid docs

Reply:
⚡️ ⚡️